libs/core/coroutines/include/hpx/coroutines/detail/context_base.hpp

```diff
@@ -149,6 +151,11 @@ namespace hpx { namespace threads { namespace coroutines { namespace detail {
             return m_state == ctx_exited;
         }
 
+        void init()
+        {
+            base_type::init();
```

Comment:
Don't we still need a `this->`, this type is a template after all?

Reply:
Apparently not.